content/features/featuresDeepDive/input/virtualJoysticks.md

```diff
@@ -35,3 +35,7 @@ To create a custom virtual joystick and modify it to suit specific use cases, th
 <Playground id="#C6V6UY#5" title="Custom Joystick Example" description="Simple example showing how to add a Custom Joystick to your scene." image="/img/playgroundsAndNMEs/divingDeeperVirtualJoystick2.jpg"/>
 
 As seen in this example, no overlay canvas is used so other GUI elements will continue to function and the visuals can be modified/positioned if needed.
+
+You can also create a custom input for a camera. For example, here is a joined input suitable for FPS games with a movement joystick on the left and swipe-pan area on the right:

```

Comment:
nit: Right side is rotating (looking) instead of panning (strafing)

Reply:
```suggestion
You can also create a custom input for a camera. For example, here is a joined input suitable for FPS games with a movement joystick on the left and swipe-rotation area on the right:
```

😅 I am coming here from being a camera operator where panning is left-right rotation without movement.
Okay, fixed.

![image](https://github.com/BabylonJS/Documentation/assets/66574145/83ed1d83-e21c-4236-b2da-b852fe86e013)